chain/events/filter/event.go

```diff
@@ -319,11 +314,18 @@ func (m *EventFilterManager) Apply(ctx context.Context, from, to *types.TipSet)
 		load:  m.loadExecutedMessages,
 	}
 
-	// TODO: could run this loop in parallel with errgroup if there are many filters
+	tsAddressResolver := m.createTipSetCachedAddressResolver()
+
+	g, ctx := errgroup.WithContext(ctx)
 	for _, f := range m.filters {
-		if err := f.CollectEvents(ctx, tse, false, m.AddressResolver); err != nil {
-			return err
-		}
+		f := f
```

Comment:
Redundant now that this module depends on Go [1.22](https://github.com/filecoin-project/lotus/blob/ffe5b280804cea2a3bf1500da922b1811635d693/go.mod#L3).
```suggestion
```